app/metrics.yaml

```diff
@@ -1856,3 +1856,56 @@ installation:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+
+addons:
+  open_addons_in_settings:
+    type: event
+    description: >
+      A user accessed "Add-ons" from the Settings
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/6174
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/8318
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  open_addon_in_toolbar_menu:
+    type: event
+    description: >
+      A user interacted with an installed add-on in the toolbar menu
+    extra_keys:
+      addon_id:
+        description: "The id of the add-on that was interacted with in the toolbar menu"
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/6174
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/8318
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  installed_addons:
```

Comment:
Instead of a string list, can we change this to `addons_installed` that is a counter with the number of add-ons installed?